packages/api/portfolioDrafts/application/createApplicationStep.ts

```diff
@@ -0,0 +1,146 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { Application } from "../../models/Application";
+import { ApplicationStep, ValidationMessage } from "../../models/ApplicationStep";
+import { Environment } from "../../models/Environment";
+import { APPLICATION_STEP } from "../../models/PortfolioDraft";
+import { dynamodbDocumentClient as client } from "../../utils/dynamodb";
+import {
+  DATABASE_ERROR,
+  NO_SUCH_PORTFOLIO_DRAFT,
+  PATH_PARAMETER_REQUIRED_BUT_MISSING,
+  REQUEST_BODY_EMPTY,
+  REQUEST_BODY_INVALID,
+} from "../../utils/errors";
+import { ApiSuccessResponse, SuccessStatusCode, ValidationErrorResponse } from "../../utils/response";
+import { isApplicationStep, isValidJson, isValidUuidV4 } from "../../utils/validation";
+export interface ApplicationValidationError {
+  applicationName: string;
+  invalidParameterName: string;
+  invalidParameterValue: string;
+  validationMessage: ValidationMessage;
+}
+
+/**
+ * Submits the Application Step of the Portfolio Draft Wizard
+ *
+ * @param event - The POST request from API Gateway
+ */
+export async function handler(event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> {
+  const portfolioDraftId = event.pathParameters?.portfolioDraftId;
+  if (!portfolioDraftId) {
+    return PATH_PARAMETER_REQUIRED_BUT_MISSING;
+  }
+  if (!isValidUuidV4(portfolioDraftId)) {
+    return NO_SUCH_PORTFOLIO_DRAFT;
+  }
+  if (!event.body) {
```

Comment:
I think you should use `isBodyPresent` from `validation.ts` for this instead of `!event.body`

Reply:
Will check that out.

Reply:
Fixed in 75ba11cb67aa4f9d5590d80382cfc19160c97ab4